distributed/comm/tests/test_ucx.py

```diff
@@ -169,22 +171,13 @@ def test_ucx_deserialize():
         lambda cudf: cudf.DataFrame([1]).head(0),
         lambda cudf: cudf.DataFrame([1.0]).head(0),
         lambda cudf: cudf.DataFrame({"a": []}),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": [1.0]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": [1]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
+        lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
+        lambda cudf: cudf.DataFrame({"a": [1.0]}).head(0),
+        lambda cudf: cudf.DataFrame({"a": [1]}).head(0),
         lambda cudf: cudf.DataFrame({"a": [1, 2, None], "b": [1.0, 2.0, None]}),
         pytest.param(
             lambda cudf: cudf.DataFrame({"a": ["Check", "str"], "b": ["Sup", "port"]}),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
+            marks=pytest.mark.skip(reason="This test segfaults for some reason. So skip running it entirely."),
```

Comment:
This test actually segfaults for me every time. I've marked it as skipped and updated the reason. Still trying to find a reproducer that doesn't involve Dask to report to cuDF. Thoughts on this would be welcome 🙂